WebContent/jsp/common/allImports.jsp

```diff
@@ -0,0 +1,24 @@
+
+
+
+<!-- import: CSS -->
+<!-- plugin css-->
+<link href="./plugins/bootstrap.css" rel="stylesheet" />
+
+<!-- app css -->
+<link href="./assets/css/styles.css" rel="stylesheet" />
+
+
+<!-- import: JS -->
+<!-- plugin js-->
+<script type="text/javascript" src="./plugins/jquery.2.2.3.min.js"></script>
+<script type="text/javascript" src="./plugins/bootstrap.min.js"></script>
+
+
+<!-- app js -->
+<script>
+	var appContextPath = "${pageContext.request.contextPath}";
+</script>
+

```

Comment:
after that, **appContextPath**  (javascript variable) can be used anywhere in your javascript ajax